README.md

```diff
@@ -0,0 +1,40 @@
+# Python Template
+
+This repository contains various templates for as many python projects as we could. We currently only have a template for  general purpuse python project/library.
+
+We also planed some other specialized templates like a django template wich should be comming soon.
+
+## How to use ?
+
+First simply click on the "use this template" button that should be somewhere in the UI :).
+We decided to use `anaconda` with `poetry` for the dependcies management if you never used them before you'll see how much it will change your life.
```

Comment:
```suggestion
We decided to use `anaconda` with `poetry` for the dependencies management if you never used them before you'll see how much it will change your life.
```